features/dynamic-bundle-discounts.md

```diff
@@ -15,61 +15,72 @@ This documentation explains the process of setting up **Nosto - Dynamic Bundles*
 
 ### Nosto Dynamic Bundle template
 
-A new Nosto JavaScript API **addBundleToCartWithDiscount** has been introduced.
-
-In order to use this new API, invoke the API from the dynamic bundle template with the following parameters,
+Nosto JavaScript API **addBundleToCartWithDiscount** can be used to add the selected bundle to the cart with the following parameters:
 
 1. Products selected in the bundle while adding to cart (_items parameter_)
 2. Discount (type & value) configuration (_discount parameter_)
 3. All the product IDs from the bundle (both selected and unselected) (_bundleProducts parameter_)
+4. A hash value that is provided with the template out of the box (_hash parameter_)
+
+Expanding _items parameter_, it should contain the following fields:
+
+1. `productId` - Product ID of the variant selected in the bundle
+2. `variantId` - ID of the selected variant in the bundle
+
+**Note:**\
+At present, Nosto supports only 1 quantity of each product in a bundle. So, quantity of selected product/variant defaults to 1
```

Comment:
It makes sense to set the default value of `quantity` to 1 but I think we should allow someone to set the quantity.

Reply:
As discussed on a call, it makes sense to not include the quantity since Dynamic Bundles doesn't currently support quantities.